staging/src/kubevirt.io/api/instancetype/v1alpha2/types.go

```diff
@@ -77,10 +77,16 @@ type VirtualMachineClusterInstancetypeList struct {
 type VirtualMachineInstancetypeSpec struct {
 
 	// Required CPU related attributes of the instancetype.
-	CPU CPUInstancetype `json:"cpu"`
+	// +optional
+	CPU *CPUInstancetype `json:"cpu,omitempty"`
 
 	// Required Memory related attributes of the instancetype.
-	Memory MemoryInstancetype `json:"memory"`
+	// +optional
+	Memory *MemoryInstancetype `json:"memory,omitempty"`
+
+	// Resources describes the Compute Resources required by this vmi
```

Comment:
`Resources describes the Compute Resources required by this instance type`

Can you also highlight that this is mutually exclusive with the above `Memory` and `CPU` attributes?